### PR TITLE
Use Systems and remove dependency on SemialgebraicSets

### LINKED ISSUE #2: Remove systems.jl and use the package Systems.jl
The package [Systems.jl](https://github.com/JuliaReach/Systems.jl) can now replace the file `src/system.jl`.
cc @schillic @mforets

### PR DESCRIPTION
**Pull request by blegat**
Closes #2
cc @mforets @schillic

### CODE REVIEW

File: src/HybridSystems.jl [hunk 1]
```diff
@@ -29,18 +60,16 @@ A hybrid system modelled as a hybrid automaton.
 - `switchings` -- vector of switchings
 - `ext`        -- dictionary that can be used by extensions
 """
-struct HybridSystem{A, S, I, G, R, W} <: AbstractHybridSystem
+struct HybridSystem{A, S, R, W} <: AbstractHybridSystem
```
Comment (mforets) [inline src/HybridSystems.jl]: `invariants` and `guards` can be removed from the docstring too.

**Comment by blegat** [comment]
@mforets Glad to hear that DiscreteIdentitySystem is welcome in Systems.jl :)

File: src/HybridSystems.jl (hunk 1 above)
Comment (blegat) [reply src/HybridSystems.jl]: Woops :-P

**Comment by schillic** [comment]
Meta comment: I learned two things from this PR. 👍
* Write functions for tests that you then call with different arguments.
* Github recognizes the "closes" keyword.

**Comment by blegat** [comment]
The first one is something we use extensively in [MOIT](https://github.com/JuliaOpt/MathOptInterfaceTests.jl) :-P

**Pull request merged**